commandline/src/main/java/tech/pegasys/web3signer/commandline/Web3SignerBaseCommand.java

```diff
@@ -180,6 +180,11 @@
       description = "Enable access logs (default: ${DEFAULT-VALUE})")
   private final Boolean accessLogsEnabled = false;
 
+  @Option(
+      names = {"--aws-cache-maximum-size"},
+      description = "Maximum cache size for AWS Secrets Manager (default: ${DEFAULT-VALUE})")
```

Comment:
Think description needs to mention that this to cache the connections. When I read this as it is it sounds like we are caching the secrets.
Perhaps something like "Maximum number of connections to cache to the AWS secrets manager."